src/ui/About.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import LiveSplitIcon from "../assets/icon_small.png";
+
+import "../css/About.scss";
+
+export class About extends React.Component {
+    public render() {
+        return (
+            <div className="about">
+                <div className="livesplit-title">
+                    <span className="livesplit-icon">
+                        <img src={LiveSplitIcon} alt="LiveSplit Logo" />
+                    </span>
+                    <div className="title-text">LiveSplit One</div>
+                </div>
+                <p className="build-version">Version: {COMMIT_HASH} ({BUILD_DATE})</p>
```

Comment:
I feel like we should make this a clickable link that links to the specific commit.